GOVERNANCE.md

```diff
@@ -170,20 +183,23 @@ yourself from the MAINTAINERS file.
 Similar to the procedure for adding new maintainers, existing maintainers can
 be removed from the list if they do not show significant activity on the
 project. Periodically, the maintainers review the list of maintainers and their
-activity over the last three months.
+activity over the last year.
```

Comment:
do we need regular date to review the list?

Reply:
There is a lot that is already vague and not sure very helpful by this language. Obviously we aren't doing this today and never have been since this was originally written pretty early on. "After a release" makes sense, but I would prefer to go through the process and document what works rather than mandating a process.